srcpkgs/Orthanc/files/orthanc/run

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+OPTS=""
+[ -r conf ] && . ./conf
+
+: ${USER:="_orthanc"}
+
+: ${CONFIGFILE:="/etc/orthanc/Configuration.json"}
+[ -r $CONFIGFILE ] || exit 1
+
+: ${ORTHANCSTORAGE:="/var/lib/OrthancStorage"}
+if [ ! -e $ORTHANCSTORAGE ]
```

Comment:
You should be quoting your use of these variables. Users might modify the `conf` file and specify spaces in the paths.